.github/workflows/updater.yml

```diff
@@ -0,0 +1,60 @@
+---
+name: Updater
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  source-build:
+    name: Install, Build & Update
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+          - 'debian:10'
+          - 'fedora:33'
+          - 'ubuntu:20.10'
+          - 'alpine:3.13'
+          - 'archlinux:latest'
+          - 'centos:8'
+          - 'clearlinux:latest'
+          - 'opensuse/tumbleweed:latest'
+    runs-on: ubuntu-latest
```

Comment:
I am a bit confused. How do you specify that the pipeline will run using a different distribution? shouldn't the **runs-on** be something like ${{ matrix.distro }}?

Reply:
Actually, I somehow missed that we’re not handling the per-distro runs correctly. Nice catch.

@kaskavel To actually run cross-distro, we need to run the code to be tested inside a Docker container. So the install, script modifications, and updater run need to be bundled into a single script that can then run inside a Docker container.

Reply:
This issue has now been addressed so feel free to take another look!